tron/bin/check_tron_jobs.py

```diff
@@ -99,9 +117,15 @@ def compute_check_result_for_job_runs(client, job, job_content):
     elif last_state == State.WAITING_FOR_FIRST_RUN:
         prefix = "OK: The job is 'new' and waiting for the first run"
         status = 0
+    elif last_state == State.SKIPPED:
+        prefix = "OK: The last job run was skipped"
+        status = 0
     elif last_state == State.STUCK:
         prefix = "WARN: Job exceeded expected runtime or still running when next job is scheduled"
         status = 1
+    elif last_state == State.CANCELLED:
```

Comment:
I think we still want to ignore cancelled runs. 
If a job was disabled on some day, they probably don't need an alert that it didn't run.
And for the non-precious case, if there's a cancelled run after a successful run, I think we should still consider the job OK.

Reply:
I agree

Reply:
Hold on, this will still be incorrect. Imagine one run is failed, and the next is cancelled. We want the check to fail.

What about changing WAITING_FIRST_RUN to something like NO_RUN_TO_CHECK: no job run to check ? I think that captures both the non-precious case (we just have a scheduled run, nothing to check yet) and the precious case (there's no run for that interval that we need to check, because the only one for that day is cancelled).

Reply:
Ah, okay. I agree with that. I'll change it.